CMakeLists.txt

```diff
@@ -291,11 +276,10 @@ function(create_library)
     target_link_libraries(${PROJECT_NAME} PUBLIC OpenMP::OpenMP_CXX)
   endif()
 
-  target_link_libraries(${PROJECT_NAME} PUBLIC proxsuite-nlp::proxsuite-nlp)
```

Comment:
Since aligator use directly proxsuite-nlp it's better to set back this line

Reply:
Done !